core/src/main/java/me/mykindos/betterpvp/core/utilities/UtilPlayer.java

```diff
@@ -74,6 +78,19 @@ public static List<KeyValue<Player, EntityProperty>> getNearbyPlayers(Player pla
         return fetchNearbyEntityEvent.getEntities();
     }
 
+    public static boolean isPlayerFriendly (Player original, Player target) {
+        return isPlayerRelation(original, target, EntityProperty.FRIENDLY, EntityProperty.ENEMY);
```

Comment:
See below comment, could be changed so that getPlayerRelation returns EntityProperty, then just check if getPlayerRelation(original, target) == EntityProperty.FRIENDLY

Reply:
implemented